new-dti-website/components/apply/ApplicationTimeline.tsx

```diff
@@ -16,14 +16,14 @@ type TabProps = {
 };
 
 const Tab: React.FC<TabProps> = ({ isSelected, text, onClick }) => (
-  <div
+  <button
     className={`flex items-center lg:px-5 lg:py-4 md:px-4 md:py-3 xs:px-2 md:rounded-t-xl xs:rounded-t-lg ${
       isSelected ? 'bg-[#FEFEFE] text-[#A52424]' : 'bg-[#7E2222CC] text-[#FEFEFE]'
     } hover:cursor-pointer md:h-min xs:h-full`}
     onClick={onClick}
   >
     <p className="font-bold lg:text-lg md:text-[13px] xs:text-[10px]">{text}</p>
-  </div>
+  </button>
```

Comment:
since these are tabs, I think we should mention their role using the ARIA `tab` role. It helps screen readers clearly label what this element is.

So you can add `role="tab"` to the `<button>` elements, and then `role="tabpanel"` to the actual tab panel contents.

[MDN has good documentation for this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role). And make sure to check out the ["Keyboard interactions" section](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#keyboard_interactions) for the correct behavior for the keyboard :)